### PR TITLE
CLXP-114 Enhancement Button Link component to have it own hover state

### PR DESCRIPTION
**Pull request by HuyIT9898**
Purpose: Enhancement Button Link component to have it own hover state

ticket: https://go1web.atlassian.net/browse/CLXP-114

review app: https://6628d2e0fd4aa63f3be2e1ab-kubwsfxkvp.chromatic.com/

### CODE REVIEW

File: packages/@coorpacademy-components/src/molecule/learner-skill-card/index.js [hunk 1]
```diff
@@ -49,10 +44,12 @@ const LearnerSkillCard = (props, context) => {
 
   const buttonExploreProps = {
```
Comment (adamska27) [inline packages/@coorpacademy-components/src/molecule/learner-skill-card/index.js:45]: can you do the same thing for the learning-priority-card? It uses the same old logic

I merged the pull request, so you maybe need to rebase,

thanks

File: packages/@coorpacademy-components/src/atom/button-link/index.tsx [hunk 2]
```diff
@@ -74,10 +82,28 @@ const ButtonLink = (props: ButtonLinkProps) => {
...
+            backgroundColor: hoverBackgroundColor,
+            color: hoverColor
+          }
+        : null)
+    };
+  }, [hoverBackgroundColor, hoverColor, hovered, customStyle]);
+
   if (link) {
     return (
       <Link
```
Comment (adamska27) [inline packages/@coorpacademy-components/src/atom/button-link/index.tsx:109]: maybe, it' a good idea to enable the same logic for the Link component too ?

File: packages/@coorpacademy-components/src/molecule/learner-skill-card/index.js (hunk 1 above)
Comment (HuyIT9898) [reply packages/@coorpacademy-components/src/molecule/learner-skill-card/index.js:45]: Thanks Adem, I've updated for it too!

File: packages/@coorpacademy-components/src/atom/button-link/index.tsx (hunk 2 above)
Comment (HuyIT9898) [reply packages/@coorpacademy-components/src/atom/button-link/index.tsx:109]: Ah, the Link itself already have hover state logic: https://github.com/CoorpAcademy/components/blob/master/packages/%40coorpacademy-components/src/atom/link/index.js#L9

**Review by adamska27 — APPROVED** [review]

**Pull request merged**